updates/info.json

```diff
@@ -1 +1 @@
-{"latest": 1, "updates": {"1": 4}}
+{"latest": 1, "updates": {"1": 12}}
```

Comment:
Shouldn't this be 14? So 28 lines divided by 2? (This is how the script woudl also auto complete it).

Reply:
The token swaps take 4 lines so is [28-(2*4)]/2 + 2

Reply:
Hmm but each token swap is two changes. You are (1) modifying an existing asset and (2) adding another asset.